versions/3.0.md

```diff
@@ -884,49 +885,50 @@ A request body with a referenced model definition.
           "$ref": "#/definitions/User"
         },
       "examples": [
-        { "$ref": 'http://foo.bar#/examples/address-example.json'}
+        { "$ref": "http://foo.bar#/examples/address-example.json"}
       ]
     },
     "application/xml" : {
        "schema": {
         "$ref": "#/definitions/User"
       },
       "examples": [
-        { "$ref": 'http://foo.bar#/examples/address-example.xml'}
+        { "$ref": "http://foo.bar#/examples/address-example.xml"}
       ]
     },
     "text/plain" : {
       "examples": [
-        { "$ref": 'http://foo.bar#/examples/address-example.txt'}
+        { "$ref": "http://foo.bar#/examples/address-example.txt"}
       ]
     },
     "*": {
       "example": {
-        $ref: "http://foo.bar#/examples/address-example.whatever"
+        "$ref": "http://foo.bar#/examples/address-example.whatever"
       }
+    }
   }
 }
 ```
 
 ```yaml
 description: user to add to the system
-content: 
-  'application/json':
+content:
+  application/json:
     schema:
-      $ref: '#/definitions/User'
+      "$ref": "#/definitions/User"
```

Comment:
I'd avoid quoting keys unless necessary (for example, in case of numbers).
For consistency, we should use single quotes where possible in the YAML values (following a recommendation from YAML's author).